### PR TITLE
Add 1.0.3 release notes

### PR DESCRIPTION
**Pull request by mandarjog**
@hklai

### CODE REVIEW

File: content/about/notes/1.0.3/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+weight: 89
+icon: notes
+---
+
+This release addresses some critical issues found by the community when using Istio 1.0.2. 
+This release note describes what's different between Istio 1.0.2 and Istio 1.0.3.
+
+{{< relnote_links >}}
+
+## Behavior Changes
```
Comment (geeknoid) [inline content/about/notes/1.0.3/index.md]: Lowercase c

File: content/about/notes/1.0.3/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Networking
+
+- `mTLS` Permissive mode is enabled by default.
+ 
+- Major performance improvement in Pilot by incremental EDS.
+
+- Default trace sampling is set to `1%`.
+
+## Policy and Telemetry
```
Comment (geeknoid) [inline content/about/notes/1.0.3/index.md]: Lowercase T

File: content/about/notes/1.0.3/index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+- Mixer (`istio-telemetry`) supports load shedding based on request rate and expected latency.
+
+- Mixer client (`istio-policy`) supports `FAIL_OPEN` setting.
+
+- Istio Performance dashboard added to grafana.
```
Comment (geeknoid) [inline content/about/notes/1.0.3/index.md]: Grafana

File: content/about/notes/1.0.3/index.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+- All `ValueTypes` supported by OOP adapters.
+
+- Reduced `istio-telemetry` CPU by `10%`.
```
Comment (geeknoid) [inline content/about/notes/1.0.3/index.md]: Should be 10% without the backticks
CPU -> CPU usage

File: content/about/notes/1.0.3/index.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+- Eliminated `stasd-to-prometheus` deployment. Prometheus directly scrapes from `istio-proxy`.
```
Comment (geeknoid) [inline content/about/notes/1.0.3/index.md]: Should that be statsd?

File: content/about/notes/1.0.3/index.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+{{< relnote_links >}}
+
+## Behavior Changes
+
+- [Validating Webhook](/help/ops/setup/validation) is now mandatory. Disabling it may result in Pilot crashes.
+
+- [ServiceEntry](/docs/reference/config/istio.networking.v1alpha3/#ServiceEntry) does not allow wildcard (`*`) DNS resolution. The API has never allowed this, however `ServiceEntry` was erroneously excluded from validation in the previous release.
+
+- Core dump path for `istio-proxy` has changed to `/var/lib/istio`.
```
Comment (geeknoid) [inline content/about/notes/1.0.3/index.md]: The core dump

File: content/about/notes/1.0.3/index.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+## Networking
+
+- `mTLS` Permissive mode is enabled by default.
```
Comment (geeknoid) [inline content/about/notes/1.0.3/index.md]: mTLS -> Mutual TLS
Can we give a link here?

File: content/about/notes/1.0.3/index.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+ 
+- Major performance improvement in Pilot by incremental EDS.
```
Comment (geeknoid) [inline content/about/notes/1.0.3/index.md]: No need to mention EDS in user docs. Can we explain this more?

Pilot performance has been greatly enhanced, reducing the time it takes to do X and enabling higher scaling.

File: content/about/notes/1.0.3/index.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+- Default trace sampling is set to `1%`.
```
Comment (geeknoid) [inline content/about/notes/1.0.3/index.md]: Link to doc on trace sampling?

File: content/about/notes/1.0.3/index.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+## Policy and Telemetry
+
+- Mixer (`istio-telemetry`) supports load shedding based on request rate and expected latency.
```
Comment (geeknoid) [inline content/about/notes/1.0.3/index.md]: now supports

File: content/about/notes/1.0.3/index.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+- Mixer client (`istio-policy`) supports `FAIL_OPEN` setting.
+
+- Istio Performance dashboard added to grafana.
+
+- All `ValueTypes` supported by OOP adapters.
```
Comment (geeknoid) [inline content/about/notes/1.0.3/index.md]: No need for this item in customer-facing release notes.

**Comment by geeknoid** [comment]
Note that once this goes in, it will need to be cherry picked into the release-1.0 branch in order to appear on istio.io

**Comment by geeknoid** [comment]
As part of this PR, please also include a one-line blog post announcing the release. Just clone the 1.0.2 one.

Thanks.

**Comment by geeknoid** [comment]
Any ETA? We need to get these notes done so we can send the announcement to the mailing list and twitter.

**Comment by mandarjog** [comment]
In the AM today.

Using "iPhone 9";

File: content/about/notes/1.0.3/index.md (hunk 5 above)
Comment (mandarjog) [reply content/about/notes/1.0.3/index.md]: `stasd-to-prometheus` is the specific thing we eliminated.

**Comment by mandarjog** [comment]
@geeknoid PTAL

File: content/about/notes/1.0.3/index.md (hunk 5 above)
Comment (geeknoid) [reply content/about/notes/1.0.3/index.md]: I mean the spelling "stasd" -> "statsd"

**Comment by geeknoid** [comment]
Looks good. So should "stasd" be changed to "statsd"?

**Comment by geeknoid** [comment]
I'll merge this to the 1.0 branch

**Pull request merged**